Firebase/Auth/Source/FIRAuth.m

```diff
@@ -512,46 +512,59 @@ - (void)signInWithEmail:(NSString *)email
                password:(NSString *)password
              completion:(FIRAuthResultCallback)completion {
```

Comment:
Any reason not to call `signInAndRetrieveDataWithEmail:password:completion:`, just like `signInWithCredential:completion:` delegates to `signInAndRetrieveDataWithCredential:completion:`?

Reply:
Done thank!